addon/adapters/rest.js

```diff
@@ -1279,7 +1279,7 @@ if (isEnabled('ds-improved-ajax')) {
 
         case 'findHasMany':
         case 'findBelongsTo':
-          let url = this.buildURL(type.modelName, id, null, requestType);
+          let url = this.buildURL(type.modelName, id, snapshots, requestType);
```

Comment:
thanks, I am still confused about when snapshot vs snapshots is passed in here. 

Is `snapshot` always a `DS.Snapshot` and this `snapshots` always `DS.SnapshotRecordArray`?